warehouse/upload.go

```diff
@@ -85,13 +85,13 @@ type UploadJobFactory struct {
 type UploadJob struct {
 	app                  app.App
 	ctx                  context.Context
-	dbHandle             *sqlmiddleware.DB
+	db                   *sqlmiddleware.DB
 	destinationValidator validations.DestinationValidator
 	loadfile             *loadfiles.LoadFileGenerator
 	tableUploadsRepo     *repo.TableUploads
 	recovery             *service.Recovery
 	whManager            manager.Manager
-	pgNotifier           *pgnotifier.PGNotifier
+	notifier             *pgnotifier.PGNotifier
```

Comment:
Is this used at all? :thinking:

Reply:
Good catch. It's not being used.

Reply:
Curious, do you find it via some static check?